python/mlc_chat/cli/delivery.py

```diff
@@ -110,6 +110,8 @@ def _run_quantization(
             logger.info("[MLC] Processing in directory: %s", output_dir)
             # Required arguments
             cmd = [
+                "python",
```

Comment:
nit: use `[sys.executable]((https://docs.python.org/3/library/sys.html#sys.executable))` instead

Reply:
Fixed in https://github.com/mlc-ai/mlc-llm/pull/1510!

Reply:
Ahh I didn't see https://github.com/mlc-ai/mlc-llm/pull/1508